doc/Makefile

```diff
@@ -154,4 +154,4 @@ doctest:
 	      "results in $(BUILDDIR)/doctest/output.txt."
 s3:
 	aws s3 sync _build/html s3://www.glueviz.org/en/`git symbolic-ref -q --short HEAD`
```

Comment:
Ah @astrofrog I think this is the issue. We also want to change the backticks here to $TRAVIS_BRANCH, otherwise the docs are uploaded directly to glueviz.org/en instead of glueviz.org/en/branch. Sorry for not catching this